File: src/java/eu/sig/training/ch03/withmapandtypes/BelgianFlag.java
package eu.sig.training.ch03.withmapandtypes;

import java.awt.Color;
import java.util.Arrays;
import java.util.List;

public class BelgianFlag implements Flag {
    public List<Color> getColors() {
        return Arrays.asList(Color.BLACK, Color.RED, Color.YELLOW);
    }
}

File: src/csharp/eu/sig/training/ch03/withmapandtypes/BelgianFlag.cs
using System.Drawing;
using System.Collections.Generic;

namespace eu.sig.training.ch03.withmapandtypes
{

    public class BelgianFlag : IFlag
    {
        public IList<Color> Colors
        {
            get
            {
                return new List<Color> { Color.Black, Color.Red, Color.Yellow };
            }
        }
    }
}
